libs/@blockprotocol/graph/src/codegen/context/postprocess.ts

```diff
@@ -30,14 +35,66 @@ export type IdentifierSources =
   | { locallyImportable: true; source: IdentifierSource }
   | { locallyImportable: false; source: IdentifierSource[] };
 
+const externalIdentifiersToSource: Record<
+  IdentifierForExternalImport,
+  IdentifierSources
+> = {
+  Entity: {
+    locallyImportable: true,
+    source: {
+      kind: "external",
+      definingPath: "@local/hash-graph-sdk/entity",
+    },
+  },
+  LinkEntity: {
+    locallyImportable: true,
+    source: {
+      kind: "external",
+      definingPath: "@local/hash-graph-sdk/entity",
+    },
+  },
+  Confidence: {
+    locallyImportable: true,
+    source: {
+      kind: "external",
+      definingPath: "@local/hash-graph-types/entity",
+    },
+  },
```

Comment:
Ideally we'd have this exported from the HASH graph client. I had to add this because it was the simplest way of having a type we can import and not be duplicated in the generated types (the underlying library we are using to generate them doesn't allow things like `PropertyProvenance["confidence"]`)